app/javascript/react/components/RideShowTile.js

```diff
@@ -11,10 +11,26 @@ const RideShowTile = props => {
     })
   }
 
+  let roundToOneDeci = (average) => {
+    return Math.round(average * 10) / 10
+  }
+
+  let ratingTotal = 0
+  let ratingAvg = "N/A"
+
+  if (props.rideReviews.length > 0) {
+    props.rideReviews.forEach(review => {
+      ratingTotal += review.rating
+    })
+    ratingAvg = roundToOneDeci((ratingTotal / props.rideReviews.length))
+
+  }
+
```

Comment:
Again, this would be more ideal to populate on the backend, but I will allow it here.